README.md

```diff
@@ -509,6 +509,16 @@ const weakMap = new WeakMap();
 weakMap.set(record, true);
 ```
 
+#### WeakSet
+
+```js
+const record = #{ a: 1, b: 2 };
+const weakSet = new WeakSet();
+
+// TypeError: Can't add a Record to a WeakMap
```

Comment:
```suggestion
// TypeError: Can't add a Record to a WeakSet
```

Reply:
That's what you get for copy pasting and editing, instead of typing everything yourself.

Fixed this